src/reanimated2/mappers.ts

```diff
@@ -6,18 +6,18 @@ import { isSharedValue } from './isSharedValue';
 
 const IS_JEST = isJest();
 
-type Mapper = {
+type Mapper<T> = {
   id: number;
   dirty: boolean;
   worklet: () => void;
-  inputs: SharedValue<any>[];
-  outputs?: SharedValue<any>[];
+  inputs: unknown[];
```

Comment:
somewhere above it was `Record<string, unknown>`, no?

Reply:
[203e122](https://github.com/software-mansion/react-native-reanimated/pull/5363/commits/203e12242a33d3199c7060fa9ec926763d785c58)